Why can't bindgen find libclang.dll?

Why can't bindgen find libclang.dll? 
```
D:\3rdparty\reaper_extension_sdk\jmde>bindgen reaper_plugin_functions.h -o reaper.rs
thread 'main' panicked at 'Unable to find libclang: "the `libclang` shared libra
ry at D:\\Program Files\\LLVM\\bin\\libclang.dll could not be opened: %1 is not
a valid Win32 application. (os error 193)"', libcore\result.rs:945:5
stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: BaseThreadInitThunk
  15: RtlInitializeExceptionChain
  16: RtlInitializeExceptionChain
```

Btw, the file "D:\Program Files\LLVM\bin\libclang.dll" exists and has version 4.0.0.0.
An earlier version of bindgen cli was working without problems on this system.

Thanks for the quick reply. I tried version 0.30.0 (`cargo install -f --vers 0.30.0 bindgen`, it seems to load libclang.dll but fails on C++ constructs.. Do I have to pass a flag to enable C++ support?
```
D:\3rdparty\reaper_extension_sdk\jmde>bindgen reaper_plugin_functions.h -o reaper.rs
./reaper_plugin.h:149:1: error: unknown type name 'class'
./reaper_plugin.h:149:26: error: expected ';' after top level declarator
./reaper_plugin.h:174:1: error: unknown type name 'class'
./reaper_plugin.h:174:21: error: expected ';' after top level declarator
./reaper_plugin.h:204:3: error: unknown type name 'MIDI_eventlist'
./reaper_plugin.h:248:3: error: unknown type name 'MIDI_eventlist'
./reaper_plugin.h:258:1: error: unknown type name 'class'
./reaper_plugin.h:258:17: error: expected ';' after top level declarator
./reaper_plugin.h:302:3: error: unknown type name 'bool'
./reaper_plugin.h:309:3: error: unknown type name 'PCM_source'
./reaper_plugin.h:375:3: error: unknown type name 'PCM_source'
./reaper_plugin.h:376:3: error: unknown type name 'PCM_source'
./reaper_plugin.h:389:1: error: unknown type name 'class'
./reaper_plugin.h:389:26: error: expected ';' after top level declarator
./reaper_plugin.h:439:3: error: unknown type name 'bool'
./reaper_plugin.h:449:1: error: unknown type name 'class'
./reaper_plugin.h:449:15: error: expected ';' after top level declarator
./reaper_plugin.h:490:3: error: unknown type name 'PCM_sink'
./reaper_plugin.h:490:91: error: unknown type name 'bool'
fatal error: too many errors emitted, stopping now [-ferror-limit=]
./reaper_plugin.h:149:1: error: unknown type name 'class', err: true
./reaper_plugin.h:149:26: error: expected ';' after top level declarator, err: t
rue
./reaper_plugin.h:174:1: error: unknown type name 'class', err: true
./reaper_plugin.h:174:21: error: expected ';' after top level declarator, err: t
rue
./reaper_plugin.h:204:3: error: unknown type name 'MIDI_eventlist', err: true
./reaper_plugin.h:248:3: error: unknown type name 'MIDI_eventlist', err: true
./reaper_plugin.h:258:1: error: unknown type name 'class', err: true
./reaper_plugin.h:258:17: error: expected ';' after top level declarator, err: t
rue
./reaper_plugin.h:302:3: error: unknown type name 'bool', err: true
./reaper_plugin.h:309:3: error: unknown type name 'PCM_source', err: true
./reaper_plugin.h:375:3: error: unknown type name 'PCM_source', err: true
./reaper_plugin.h:376:3: error: unknown type name 'PCM_source', err: true
./reaper_plugin.h:389:1: error: unknown type name 'class', err: true
./reaper_plugin.h:389:26: error: expected ';' after top level declarator, err: t
rue
./reaper_plugin.h:439:3: error: unknown type name 'bool', err: true
./reaper_plugin.h:449:1: error: unknown type name 'class', err: true
./reaper_plugin.h:449:15: error: expected ';' after top level declarator, err: t
rue
./reaper_plugin.h:490:3: error: unknown type name 'PCM_sink', err: true
./reaper_plugin.h:490:91: error: unknown type name 'bool', err: true
fatal error: too many errors emitted, stopping now [-ferror-limit=], err: true
thread 'main' panicked at 'Unable to generate bindings: ()', libcore\result.rs:9
45:5
stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: BaseThreadInitThunk
  18: RtlUserThreadStart
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,7 +14,7 @@ readme = "README.md"
 repository = "https://github.com/rust-lang-nursery/rust-bindgen"
 documentation = "https://docs.rs/bindgen"
 homepage = "https://rust-lang-nursery.github.io/rust-bindgen/"
-version = "0.37.1"
+version = "0.37.2"
 build = "build.rs"
 
 include = [
@@ -47,7 +47,7 @@ cexpr = "0.2"
 cfg-if = "0.1.0"
 # This kinda sucks: https://github.com/rust-lang/cargo/issues/1982
 clap = "2"
-clang-sys = { version = "0.22.0", features = ["runtime", "clang_6_0"] }
+clang-sys = { version = "0.23", features = ["runtime", "clang_6_0"] }
 lazy_static = "1"
 peeking_take_while = "0.1.2"
 quote = { version = "0.5", default-features = false }

